apps/chat/src/chat/chat.controller.ts

```diff
@@ -1,8 +1,14 @@
 import { Controller } from '@nestjs/common';
+import { MessagePattern, Payload } from '@nestjs/microservices';
 import { ChatService } from './chat.service';
 
 //* 채팅 서비스의 HTTP 요청/응답 처리 Controller
 @Controller()
 export class ChatController {
   public constructor(private readonly chatService: ChatService) {}
+
+  @MessagePattern({ cmd: 'create_room' })
+  public createRoom(@Payload() { roomName }: { roomName: string }) {
+    return this.chatService.createRoom(roomName);
+  }
```

Comment:
```suggestion
  @MessagePattern(MESSAGE.CHAT.CREATE_ROOM)
  public createRoom(@Payload() { roomName }: { roomName: string }) {
    return this.chatService.createRoom(roomName);
  }
```

위의 msa controller 에서도 메세지 패턴을 상수화 하여 사용하면 좋을 것 같아요!

Reply:
[6da2345](https://github.com/GrebbMe/grabbme-be/pull/35/commits/6da2345f90842a259fe09d2d7eb912c9376e2979)

위 커밋에 반영하였습니다!

Reply:
지금  message-pattern 파일에 CHAT 쪽 상수가 PUBLIC_DATA 의 자식으로 선언 되어 있습니다!
MESSAGE.CHAT.CREATE_CHAT_ROOM 으로 사용 가능하도록 변경해주세요!
(merge Conflict 도 해당 문제 때문에 발생합니다!)

Reply:
[b048d9b](https://github.com/GrebbMe/grabbme-be/pull/35/commits/b048d9bd0515dc102a897bdfb4fa61d22adad6c1)

경로수정 반영하였습니다!